tabuada.py

```diff
@@ -0,0 +1,7 @@
+
+number = int(input("Digite o numero que deseja a tabuada: "))
+multiplo = 0 
```

Comment:
typo

Reply:
typo zero ?

Reply:
change ''multiplo'' variable to an english word

Reply:
revisa de novo porra

Reply:
brincadeira s2